projects/uitdatabank/docs/errors.md

```diff
@@ -95,3 +95,11 @@ For example, you cannot update the location of an event with attendanceMode `onl
 *   **Status**: `400`
 
 The action you are trying to perform on an event cannot be done because the event has UiTPAS ticket sales. When an event has UiTPAS ticket sales its organizer, UiTPAS card system(s) and UiTPAS distribution key(s) cannot be updated because it would invalidate the existing ticket sales in UiTPAS.
+
+## image-cannot-be-main-image
+
+*   **Complete type:** `https://api.publiq.be/probs/uitdatabank/image-cannot-be-main-image`
+*   **Title**: `Image cannot be main image`
+*   **Status**: `400`
+
+The action you are trying to perform on an event or place cannot be done because the image has not been added to it. Add the image first to the event or place, so you can set it as the main image.
```

Comment:
```suggestion
When setting a main image on an event or place, the image id must be one of the images that are already linked to the event or place. Add the image first to the event or place, so you can set it as the main image.
```

The "the action you are trying to perform" text is mostly for errors that can happen in multiple scenario's, whereas this error can only happen in 1 scenario (= when setting the main image on an event/place)